sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -15104,6 +15104,27 @@ public void testInGroupByOrderByLimitOutGroupByOrderByLimit()
         .run();
   }
 
+
+  @Test
+  public void testScanAndSortCanGetSchemaFromScanQuery()
+  {
+    skipVectorize();
+    cannotVectorize();
+    msqIncompatible();
```

Comment:
Why is it `msqIncompatible()`? Seems like this query should work in MSQ.

Reply:
I've just copied an existing testcase when I created the test - removed these flags